content/blog/experiences-with-ci-cd.pt-br.md

```diff
@@ -0,0 +1,476 @@
++++
+draft = false
+date  = 2022-04-10
+
+title    = "Experiências com CI/CD e DevOps"
+subtitle = "Praticando DevOps com papel e caneta"
+
+toc = true
++++
+
+**AVISO:** Apesar da falta relativa de detalhes, esse artigo acabou tão extenso
+que fui obrigado a incluir uma tabela de conteúdo. Espero que não seja uma
+leitura cansativa, e que quem estiver lendo consiga pular para um tópico de seu
+interesse.
+
+O propósito desta postagem é dar ao leitor um resumo de como foi, para mim, se
+acostumar com e implementar soluções de _integração contínua (CI)_ e _deployment
+contínuo (CD)_ em meus projetos pessoais. Foi e ainda está sendo como uma
+rodovia esburacada, mas é mais gratificante do que achei que seria quando
+comecei.
+
+Minha atitude para com serviços, programas e plataformas de CI/CD era de que
+"não são pra mim", e algo de que só projetos grandes ou empresas, liderados por
+uma força-tarefa altamente qualificada, precisariam e tirariam proveito.
+
+Tal posição com certeza mudou quando, trabalhando diretamente com o ciclo de
+desenvolvimento de software e melhorando-o, me toquei do seu valor e sentido.
```

Comment:
Suggestion: "tomei consciência do seu valor e sentido."

Reply:
That does sound a better when you read it. Thanks! Applied in ead6a2d.